partials/tabs/projects.html

```diff
@@ -30,46 +30,44 @@
   <div id="modal1" class="modal">
     <div  class="modal-content-bear">
       <div class="col-md-4 profile">
-        <div class="project-name">
-          {{currentProject.name}}
+        <div btf-markdown="currentProject.name" class="project-name markdown">
         </div>
         <div ng-show="currentProject.requirements.length>0" class="project-detail-element  reqs-proj">
           <div class="small-heading">REQUIREMENTS</div>
-          <li class="pr-element-detail milestones" ng-repeat="req in currentProject.requirements">{{ req }}</li><br>
+          <li btf-markdown="req" class="pr-element-detail milestones markdown" ng-repeat="req in currentProject.requirements"></li><br>
         </div>
 
         <div ng-show="currentProject.initiatives.length>0" class="project-detail-element">
           <div class="small-heading">INITIATIVES</div>
-          <span class="pr-element-detail chip" ng-repeat="initiative in currentProject.initiatives">{{ initiative }}</span><br>
+          <span btf-markdown="initiative" class="pr-element-detail chip markdown" ng-repeat="initiative in currentProject.initiatives"></span><br>
```

Comment:
I don't think we need markdown for the initiatives, tags and other one-word things